aspnetcore/grpc/services.md

```diff
@@ -145,96 +145,104 @@ message SimpleRequest {
 A server streaming method gets the request message as a parameter. Because multiple messages can be streamed back to the caller, `responseStream.WriteAsync` is used to send response messages. A server streaming call is complete when the method returns.
 
 ```csharp
-public override async Task StreamingFromServer(SimpleRequest request,
-    IServerStreamWriter<SimpleResponse> responseStream, ServerCallContext context)
+public override async Task StreamingFromServer(BasicRequest request,
+    IServerStreamWriter<BasicResponse> responseStream, ServerCallContext context)
 {
     for (var i = 0; i < 5; i++)
     {
-        await responseStream.WriteAsync(new SimpleResponse());
+        await responseStream.WriteAsync(new BasicResponse());
         await Task.Delay(TimeSpan.FromSeconds(1));
     }
 }
 ```
 
-The client has no way to send additional messages or data once the server streaming method has started. Some streaming methods are designed to run forever. In this situation a client can cancel the call when they no longer need it. When cancellation happens the client sends a signal to the server and the `ServerCallContext.CancellationToken` <xref:System.Threading.CancellationToken> is raised. The token should be used on the server with async methods used so that any asynchronous work is canceled together with the streaming call, and the method exits quickly.
+The client has no way to send additional messages or data once the server streaming method has started. Some streaming methods are designed to run forever. For continuous streaming methods, a client can cancel the call when it's no longer need it. When cancellation happens the client sends a signal to the server and the [ServerCallContext.CancellationToken](xref:System.Threading.CancellationToken) is raised. The `CancellationToken` token should be used on the server with async methods so that:
```

Comment:
```suggestion
The client has no way to send additional messages or data once the server streaming method has started. Some streaming methods are designed to run forever. For continuous streaming methods, a client can cancel the call when it's no longer needed. When cancellation happens the client sends a signal to the server and the [ServerCallContext.CancellationToken](xref:System.Threading.CancellationToken) is raised. The `CancellationToken` token should be used on the server with async methods so that:
```